app/presenters/formats/answer_presenter.rb

```diff
@@ -1,5 +1,10 @@
 module Formats
   class AnswerPresenter < EditionFormatPresenter
+
+  def render_for_publishing_api(republish: false, rendering_app: "government-frontend")
```

Comment:
I was thinking more along the lines of there being a `rendering_app` method in this class, which would override the same method on the super class.
eg.
```
def rendering_app
  'government-frontend'
end
```